src/api/v1alpha1/helpers.go

```diff
@@ -54,7 +54,7 @@ func (dk *DynaKube) ActiveGateImage() string {
 	}
 
 	registry := buildImageRegistry(dk.Spec.APIURL)
-	return fmt.Sprintf("%s/linux/activegate:latest", registry)
+	return fmt.Sprintf("%s/linux/activegate:raw", registry)
```

Comment:
you defined a const for `raw` why not use it here aswell?

Reply:
why are we updating `v1alpha` helpers? these should not be used anymore

Reply:
As long as the code exists it should be maintained. If its dead code, it should be removed.